deploy/bootstrap/run.sh

```diff
@@ -551,7 +551,7 @@ cat >> ${PRIVATE}/lega.yml <<EOF
     restart: on-failure:3
     networks:
       - lega
-    image: cscfi/ega-keyserver
```

Comment:
What's the reason for not using the latest keyserver image?

Reply:
latest image requires HTTPS configuration, and did not want to deal with it now, so i reverted to using a version that works with HTTP, until i figure out how to generate the certificates for HTTPS.
Of course we can include it this PR

Reply:
Not worth the effort for now.